README.md

```diff
@@ -1,57 +1,348 @@
+  - Proposing configuration changes to route traffic over the most efficient paths between every possible ingress and egress router
+  - Trigger configuration update to actually implement the proposed optimization in the network.
+- **Validator** which is to be developed after the hackathon and validates the optimiser's configuration changes to e.g. prevent data from being sent through bottlenecks or certain paths from being overprovisioned.
 
-### After Hackathon
+## Contents
 
-- Review of fundamental concepts
-- Detect data shortages to do the optimizations
-- Implementation of a validation component which checks if it acutally makes sense to apply the suggested change
+- [Project Overview](#project-overview)
+- [Project Objectives During Hackathon](#project-objectives-during-hackathon)
+- [Getting Started](#getting-started)
+- [Related Work](#related-work)
 
-## Background
 
-### Motivation
+## Project Overview
 
-Over the past year, my colleagues and I have delved into the field of sustainable networking, focusing on the critical challenge of improving energy efficiency in computer networks.
-Our initial research revealed a significant gap: optimizing network energy efficiency through traffic engineering methods is extremely difficult due to the lack of visibility into the energy efficiency of network paths.
+Over the past year, my colleagues and I (Ramon) have delved into the field of sustainable networking, focusing on the critical challenge of improving energy efficiency in computer networks.
+Our initial research revealed a significant gap: **optimizing network energy efficiency through traffic engineering methods is extremely difficult due to the lack of visibility into the energy efficiency of network paths.**
 
 This realization led us to focus on enhancing visibility into the energy efficiency of network paths and nodes.
-After several months of research, we published a paper titled "Towards Sustainable Networking: Unveiling Energy Efficiency Through Hop and Path Efficiency Indicators in Computer Networks", which was presented at IEEE Netsoft 2024 and is now available on [IEEE Xplore](https://ieeexplore.ieee.org/document/10588907).
+After several months of research, we published a paper titled *"Towards Sustainable Networking: Unveiling Energy Efficiency Through Hop and Path Efficiency Indicators in Computer Networks"*, which was presented at IEEE Netsoft 2024 and is now available on [IEEE Xplore](https://ieeexplore.ieee.org/document/10588907).
+
+Our proof-of-concept (PoC) implementation already includes several key capabilities:
+- The collection of network telemetry data embedded in packet metadata using the [IOAM Aggregation Trace Option](https://datatracker.ietf.org/doc/html/draft-cxx-ippm-ioamaggr-02)
+- Standardized export of the collected network telemetry data to an [IPFIX](https://www.rfc-editor.org/rfc/rfc7011) collector
+- Visualization of the collected data on descriptive Grafana dashboards
```

Comment:
I would link a picture so that the user can get an idea of what the visualization dashboard looks like.

Reply:
Good idea!
I will do that.

Reply:
done